cmd/storage-datatypes.go

```diff
@@ -234,6 +234,20 @@ type FileInfo struct {
 	Checksum []byte `msg:"cs,allownil"`
 }
 
+// WriteQuorum returns expected write quorum for this FileInfo
+func (fi FileInfo) WriteQuorum() int {
+	quorum := fi.Erasure.DataBlocks
+	if fi.Erasure.DataBlocks == fi.Erasure.ParityBlocks {
+		quorum++
+	}
+	return quorum
+}
+
+// ReadQuorum returns expected read quorum for this FileInfo
+func (fi FileInfo) ReadQuorum() int {
+	return fi.Erasure.DataBlocks
+}
+
```

Comment:
Even if it isn't relevant for the current use, deletemarkers with their 50% should probably be handled here as well.